CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 ### Changed
 - Added support for deleting vectors by metadata filter. The pinecone.Index.delete() api now accepts an additional filter= parameter which takes metadata filter expression equivalent to what query() supports.
   - Internally these requests are now sent as POST requests, though the previous DELETE api is still supported.
+- Added support of querying by vector id.
```

Comment:
We have a release for 2.0.10 can you put it under a new Unreleased header?

Reply:
Didn't know that. Updateing